test/test_limiter.rb

```diff
@@ -20,6 +20,14 @@
         Prop::Limiter.disabled { Prop.throttle(:something) }.must_equal false
       end
 
+      it "supports decrement" do
+        Prop.throttle(:something)
+        Prop.count(:something).must_equal 1
+
+        Prop.throttle(:something, nil, decrement: 1)
+        Prop.count(:something).must_equal 0
+      end
```

Comment:
`it 'can decrement below 0'` ... just as docs / sanity check that nothing blows up ...